packages/terra-date-picker/tests/wdio/calendar-filter-spec.js

```diff
@@ -0,0 +1,93 @@
+Terra.describeViewports('Calendar Filter', ['tiny'], () => {
```

Comment:
@benbcai Do we need to add in time mocks for these test to run independent of the docker containers timezone?

Reply:
Yes, ideally we would need to update the other tests as well.

Reply:
I am sorry..! I am bit confused are you referring to set timezone in tests like this ..? 
```jsx
moment.tz.setDefault('America/Chicago');
```

Reply:
@supreethmr Timezone is now defined in the docker image. You can ignore this for now. We will address this by mocking the timezone in a separate PR.